third_party/terraform/website/docs/r/app_engine_standard_app_version.html.markdown

```diff
@@ -0,0 +1,72 @@
+---
```

Comment:
This file should be generated automatically, you can remove it.

Reply:
Thanks for your support, I have made the require changes. 
But I think I need to upload the hello-world.zip in the test-fixtures folder, is it possible to do that.
hello-world.zip is based on the https://github.com/GoogleCloudPlatform/nodejs-docs-samples/tree/master/appengine/hello-world/standard

I have tested by deploying using the new resource.

once again thanks for your support.

Reply:
I have one more question, what are the guidelines for implementing the resource update . So essentially the patch verb for updating the service uses fields which are not the same as create version. The patch verb actually updates the version for splitting the traffic.

Right now I am able to create and delete the version using the new resource.
But in the context of appengine version resource (https://cloud.google.com/appengine/docs/admin-api/reference/rest/v1/apps.services.versions/patch),
the patch should split the traffic between multiple versions. 

And I am trying to figure out what is the right way to implement it. Should I create a separate resource for splitting the traffic?

Reply:
`test-fixtures` files live at https://github.com/GoogleCloudPlatform/magic-modules/tree/master/third_party/terraform/utils/test-fixtures, you can add the field there.

Traffic splitting is probably something we'll need to implement as a handwritten resource. It's configured in https://cloud.google.com/appengine/docs/admin-api/reference/rest/v1/apps.services/patch, right?

That way, users will be able to create versions in their TF config and then reference those versions from a separate traffic splitting resource.

Reply:
Added the hello-world.zip, getting excited for having this PR approved. I really appericiate your guidance and support @rileykarson.
I am looking into writing the new resource for traffic splitting.

Reply:
while running the tests locally, I am getting this error
=== CONT  TestAccAppEngineStandardAppVersion_appEngineStandardAppVersionExample
Error creating bucket : googleapi: Error 400: Empty bucket name, invalid--- FAIL: TestAccAppEngineStandardAppVersion_appEngineStandardAppVersionExample (1.25s)
    testing.go:568: Step 0 error: errors during apply:

        Error: googleapi: Error 400: Empty bucket name, invalid

          on /tmp/tf-test209836460/main.tf line 13:
          (source code not available)



        Error: error creating project test-projectz422xu7bmt (tf-test-project): googleapi: Error 400: Request contains an invalid argument., badRequest. If you received a 403 error, make sure you have the `roles/resourcemanager.projectCreator` permission

          on /tmp/tf-test209836460/main.tf line 2:
          (source code not available)


FAIL
FAIL    github.com/terraform-providers/terraform-provider-google/google 1.285s
GNUmakefile:16: recipe for target 'testacc' failed

Reply:
trying to find out how to set the bucket name variable

Reply:
It looks like there have been some commits since this was posted- are you encountering the same issue still?